design/ceph/multus-network.md

```diff
@@ -90,111 +90,53 @@ interface will be added to the host network namespace for all nodes that will ru
 strategy to `OnDelete` so that the pods do not get deleted if the DaemonSet is updated while also
 ensuring that the pods will be updated on the next node reboot (or node drain).
 
-#### Mover DaemonSet and Pods
-The Rook-Ceph Operator's CSI controller also creates a second DaemonSet configured to use host
-networking. This DaemonSet also runs on all nodes that will have CSI plugin pods (and holder pods).
-Mover pods exist to "move" the multus network interface being held by the holder pod on the node
-into the host's network namespace to provide user's volumes with uninterrupted access to the Ceph
-cluster, even when the CSI driver is restarted (or updated).
-
-The mover must:
-- be a privileged container
-- have `SYS_ADMIN` and `NET_ADMIN` capabilities
-- be on the host network namespace
-- have access to the `/var/run/netns` directory
-
-In order to not leave moved interfaces dangling in the host's network namespace, mover pods must
-move interfaces back to their original namespace when CSI is being terminated. The most
-straightforward way to accomplish this is to move interfaces back when the mover is being
-terminated. If an interface is moved without user applications also being removed, this will cause
-I/O disruption. Therefore, the DaemonSet should also use the `OnDelete` update strategy so that the
-pods can be updated on node reboots (or node drains).
-
-In order to better handle unexpected corner cases that leave moved interfaces in the host network
-namespace (e.g., a mover is killed abruptly rather than gracefully terminated), instead treat "move"
-operations as a disable-and-copy operation. To do this, disable the interface in the holder pod's
-network namespace, and create a copy of the interface in the host namespace with the same MAC
-address and IP config. From the user standpoint, the interface is still "moved" because the original
-is disabled, so we keep the "mover" terminology. This merely helps Rook ensure that it is not
-accidentally losing the original information.
-
-A previous iteration of this design specified the mover application as a sidecar to Ceph CSI plugin
-pods; however, this design would mean that the mover would need to be deleted and re-created
-whenever the CSI plugin is updated, possibly resulting in I/O hangs in user pods during the update.
-Keeping the mover independent allows CSI plugin updates to happen freely without complex
-interactions between it and the mover.
+The new holder DaemonSet only contains a single container called `holder`, the container
+responsible for holding filesystem mounts and mapped block devices. It is used as a passthrough by
+the Ceph-CSI plugin pod which when mounting or mapping will use the network namespace of that holder
+pod.
+
+One task of the Holder main container is to passing the required PID into the `csi-plugin` pod. To
+solve this, the main container will create a symlink to an equivalent of `/proc/$$/ns/net` in the
```

Comment:
"main container" makes it sound like there would be more than one container?  That goes against "The new holder DaemonSet only contains a single container" above.
```suggestion
One task of the Holder container is to pass the required PID into the `csi-plugin` pod. To
solve this, the container will create a symlink to an equivalent of `/proc/$$/ns/net` in the
```